src/it/java/com/google/checkstyle/test/chapter4formatting/rule452indentcontinuationlines/IndentContinuationLinesAtLeast4SpacesTest.java

```diff
@@ -17,7 +17,7 @@
 // Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-package com.google.checkstyle.test.chapter4formatting.rule452continuationlines;
+package com.google.checkstyle.test.chapter4formatting.rule452indentcontinuationlines;
```

Comment:
corrected the package name. Input file package name and test class package name were different.